python/turbine_models/custom_models/stateless_llama.py

```diff
@@ -277,11 +278,11 @@ def forward(token0: torch.Tensor, *state0_flat):
         with open(f"{safe_name}.vmfb", "wb+") as f:
             f.write(flatbuffer_blob)
         print("saved to ", safe_name + ".vmfb")
-        exit()
+        return module_str, tokenizer
 
 
 def run_vmfb_comparison(args):
-    config = ireert.Config(args.device)
+    config = ireert.Config("local-task")
```

Comment:
local-task is a cpu specific device config. If we are trying to run on gpu we would want to provide a different device flag

Reply:
Using args.device doesn't work (i think neither cpu or llvm-cpu works).

From the error messages I got, this needs a device URI instead of a device type. I think there are IREE commands to list the available devices and maybe we could automatically pick somehow.

I'm thinking maybe we should fold run_vmfb_comparison into a test instead, so we don't have to make it work for different devices.

Reply:
yeah for runtime the cpu device flag should be either local-task or local-sync. With the runtime part of stateless_llama the default can't really work for both compile and runtime for cpu unfortunately